pkgs/tools/system/kanata/default.nix

```
@@ -0,0 +1,39 @@
+{ fetchFromGitHub
+, lib
+, libevdev
+, pkg-config
+, rustPlatform
+}:
+
+let
+  pname = "kanata";
+  version = "1.0.5";
+in
+rustPlatform.buildRustPackage {
```

There's no need to have a `let` block here.
```suggestion
rustPlatform.buildRustPackage rec {
```

---

I don't like `rec`. See https://nix.dev/anti-patterns/language

---

The reasoning for me is the same as the above. Yes, `rec` can be bad, but in a scope of a 30 LOC module it's not a big deal IMO.

---

> I don't like `rec`. See [nix.dev/anti-patterns/language](https://nix.dev/anti-patterns/language)

that link does not apply to nixpkgs packages but it is pretty much whatever. You treat extra lines of code for some bits in stack.